cart/domain/cart/cart.go

```diff
@@ -327,6 +327,20 @@ func (c Cart) SumShippingNet() domain.Price {
 	return price
 }
 
+// SumShippingNetWithDiscounts - returns net sum price of deliveries ShippingItems with discounts
+func (c Cart) SumShippingNetWithDiscounts() domain.Price {
+	prices := make([]domain.Price, 0, len(c.Deliveries))
+
+	for _, del := range c.Deliveries {
+		discount, _ := del.ShippingItem.AppliedDiscounts.Sum()
+		prices = append(prices, del.ShippingItem.PriceNet.ForceAdd(discount))
+	}
+
+	price, _ := domain.SumAll(prices...)
+
+	return price
+}
+
```

Comment:
Could you maybe provide a test showing what price you expect given a cart with items and shipping items from this function?

Reply:
Added a test as requested.